src/components/FilterTagsRender.js

```diff
@@ -0,0 +1,37 @@
+import { useHistory, useLocation } from "react-router-dom";
+import { stringify, parse } from "../utilities/queryString";
+import { toggleValueInArray } from "../utilities/filters/toggleValueInArray";
```

Comment:
The `toggleValueInArray` utility could be saved just under `utilities` as it's not really a filter